ctp2_code/ai/strategy/scheduler/scheduler.cpp

```diff
@@ -879,6 +1120,8 @@ void Scheduler::Match_Resources(const bool move_armies)
 	{
 		Goal_ptr goal_ptr        = static_cast<Goal_ptr>(*goal_iter2);
 		committed_agents_test   += goal_ptr->Get_Agent_Count();
+		if (goal_ptr->Get_Agent_Count() > 0)
+			committed_agents_test = committed_agents_test;
```

Comment:
This seems an odd statement as it says a = a; Please check what is intended.

Reply:
I meant to remove this it was just somewhere for me to put a breakpoint when trying to find why the error in the assert was occuring. I will fix this.